### PR TITLE
relationship_of_periods_of_planets_with_mass_and_semi_axes_of_orbits

### PR DESCRIPTION
**Pull request by Zulusssss**
Add law and tests.

### CODE REVIEW

File: symplyphysics/laws/astronomy/relationship_of_periods_of_planets_with_mass_and_semi_axes_of_orbits.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+first_period = Symbol("first_period", units.time)
+second_period = Symbol("second_period", units.time)
+first_semi_axis = Symbol("first_semi_axis", units.length)
+second_semi_axis = Symbol("second_semi_axis", units.length)
+first_mass = Symbol("first_mass", units.mass)
+second_mass = Symbol("second_mass", units.mass)
+
+mass_of_sun = Quantity(1.989e30 * units.kilogram)
+
+law = Eq(((mass_of_sun + first_mass) * (first_period)**2) / ((mass_of_sun + second_mass) * (second_period)**2), (first_semi_axis)**3 / (second_semi_axis)**3)
```
Comment (blackyblack) [inline symplyphysics/laws/astronomy/relationship_of_periods_of_planets_with_mass_and_semi_axes_of_orbits.py:34]:
```suggestion
law = Eq(((mass_of_sun + first_mass) * first_period**2) / ((mass_of_sun + second_mass) * second_period**2), first_semi_axis**3 / second_semi_axis**3)
```

File: symplyphysics/laws/astronomy/relationship_of_periods_of_planets_with_mass_and_semi_axes_of_orbits.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    validate_input,
+    validate_output,
+)
+
+# Description
+## Kepler's third law is as follows. The squares of the periods of rotation of the planets around the Sun are referred to
+## as cubes of the large semi-axes of the orbits of the planets. But Newton found that Kepler's third law is not entirely
+## accurate — in fact, it includes the masses of the planets and the Sun.
+
+## Law is: ((M + m1) * T1^2) / ((M + m2) * T2^2) = a1^2 / a2^2, where
```
Comment (blackyblack) [inline symplyphysics/laws/astronomy/relationship_of_periods_of_planets_with_mass_and_semi_axes_of_orbits.py:16]: Does not correspond to description.

File: test/astronomy/relationship_of_periods_of_planets_with_mass_and_semi_axes_of_orbits_test.py [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+Args = namedtuple("Args", ["second_period", "first_semi_axis", "second_semi_axis", "first_mass", "second_mass"])
+
+
+@fixture(name="test_args")
+def test_args_fixture() -> Args:
+    second_period = Quantity(1.88 * units.year)
+    first_semi_axis = Quantity(1 * units.astronomical_unit)
+    second_semi_axis = Quantity(1.524 * units.astronomical_unit)
+    first_mass = Quantity(5.9742e24 * units.kilogram)
+    second_mass = Quantity(6.39E23 * units.kilogram)
```
Comment (blackyblack) [inline test/astronomy/relationship_of_periods_of_planets_with_mass_and_semi_axes_of_orbits_test.py:19]:
```suggestion
    second_mass = Quantity(6.39e23 * units.kilogram)
```

**Review by blackyblack — APPROVED** [review]

File: symplyphysics/laws/astronomy/relationship_of_periods_of_planets_with_mass_and_semi_axes_of_orbits.py (hunk 2 above)
Comment (blackyblack) [reply symplyphysics/laws/astronomy/relationship_of_periods_of_planets_with_mass_and_semi_axes_of_orbits.py:15]: Not fixed.

Comment (Zulusssss) [reply symplyphysics/laws/astronomy/relationship_of_periods_of_planets_with_mass_and_semi_axes_of_orbits.py:15]: > Not fixed.

I don't understand what the discrepancy is with the description.

**Review by blackyblack — APPROVED** [review]

**Pull request merged**